src/api/artifacts-api.js

```diff
@@ -37,6 +41,10 @@ const fetchFeatureStoreData = (item, tab, config) => {
     url = `${url}?${labels}`
   }
 
+  if (item?.tag && !/latest/i.test(item.tag)) {
+    url = `${url === initialUrl ? `${url}?` : `${url}&`}tag=${item.tag}`
```

Comment:
```suggestion
    const delimiter = url === initialUrl ? '?' : '&'
    url = `${url}${delimiter}tag=${item.tag}`
```

But, really, instead of constructing the URL query parameters string yourself, you could just use Axios’ config `params` property.

```js
axios.get(url, {
  params: {
    a: 'b',
    c: 'd'
  }
})
// => url?a=b&c=d
```

Like you did [here](https://github.com/mlrun/ui/blob/development/src/api/jobs-api.js#L26)